mlir/include/mlir/Dialect/Tosa/IR/TosaComplianceData.h.inc

```diff
@@ -406,9 +431,9 @@ extensionComplianceMap = {
       {{Extension::fp8e4m3}, {{fp8e4m3T, fp8e4m3T}}},
       {{Extension::fp8e5m2}, {{fp8e5m2T, fp8e5m2T}}},
       {{Extension::bf16}, {{bf16T, bf16T}}}}},
-    {"tosa.dim",
-     {{{Extension::fp8e4m3}, {{fp8e4m3T}}},
-      {{Extension::fp8e5m2}, {{fp8e5m2T}}},
-      {{Extension::bf16}, {{bf16T}}}}},
+    {"tosa.variable_write",
+     {{{Extension::variable}, {{i8T}, {fp16T}, {fp32T}}}}},
+    {"tosa.variable_read",
+     {{{Extension::variable}, {{i8T}, {fp16T}, {fp32T}}}}},
```

Comment:
Some of the operators seem to be declared to ignore when populating compliance info, now that they are listed here, should the code below be updated to reflect this: https://github.com/llvm/llvm-project/pull/131208/files#diff-97cc34fa7e2a0ae27da6c64c8d0c6873ec4ca9018d9c5e9d0b7e44ff185f156dR241?

Reply:
Yes, move these opterators into the `POPULATE_PROFILE_INFO_COMMON` group.

Reply:
It turns out that VARIABLE needs to be handled specially as its `initial_value` is attribute type rather than operand type.